index.html

```diff
@@ -0,0 +1,141 @@
+            <div id="info-message-container">
+              <div id="date-message" class="lato-black dark">2015.10.15(THU) ~ 16(FRI)</div>
+              <div id="location-message" class="lato dark">@ National Museum of Korea, Art Center Nabi and more</div>
+            </div>
+            </div>
+        </div>
+      </section>
+      <section id="program">
+        <div id="program-container">
+          <div id="heading-container">
+            <h2 class="lato white section-heading" id="program-heading">Main Program</h2>
+            <div id="indicator1" class='indicator red-bg'></div>
+          </div>
+          <div id="program-cards-container">
+            <article id="pr-card1" class="pr-card">
+              <i id="pr-icon-1" class="pr-card-icon fa-solid fa-graduation-cap white"></i>
+              <h3 id="pr-heading-1" class="pr-heading lato red">Lecture</h3>
+              <p id="pr-desc-1" class="pr-desc-text white lato">
+                Lorem Ipsum is simply dummy text of the printing and typesetting industry.
+              </p>
+            </article>
+            <article id="pr-card2" class="pr-card">
+              <i id="pr-icon-2" class="pr-card-icon fa-solid fa-graduation-cap white"></i>
+              <h3 id="pr-heading-2" class="pr-heading lato red">CC Exhibition</h3>
+              <p id="pr-desc-2" class="pr-desc-text white lato">
+                Lorem Ipsum is simply dummy text of the printing and typesetting industry.
+              </p>
+            </article>
+            <article id="pr-card3" class="pr-card">
+              <i id="pr-icon-3" class="pr-card-icon fa-solid fa-graduation-cap white"></i>
+              <h3 id="pr-heading-3" class="pr-heading lato red">Forum</h3>
+              <p id="pr-desc-3" class="pr-desc-text white lato">
+                Lorem Ipsum is simply dummy text of the printing and typesetting industry.
+              </p>
+            </article>
+            <article id="pr-card4" class="pr-card">
+              <i id="pr-icon-4" class="pr-card-icon fa-solid fa-graduation-cap white"></i>
+              <h3 id="pr-heading-4" class="pr-heading lato red">Workshop</h3>
+              <p id="pr-desc-4" class="pr-desc-text white lato">
+                Lorem Ipsum is simply dummy text of the printing and typesetting industry.
+              </p>
+            </article>
+            <article id="pr-card5" class="pr-card">
+              <i id="pr-icon-5" class="pr-card-icon fa-solid fa-graduation-cap white"></i>
+              <h3 id="pr-heading-5" class="pr-heading lato red">CC Ignite</h3>
+              <p id="pr-desc-5" class="pr-desc-text white lato">
+                Lorem Ipsum is simply dummy text of the printing and typesetting industry.
+              </p>
+            </article>
+          </div>
+          <button id="join-btn" class="red-bg lato white">Join CWA Full-Stack Program</button>
+          <a id="program-link" class="lato white">SEE THE WHOLE PROGRAM</a>
```

Comment:
- [ ] [OPTIONAL] I have noticed that the `see the whole program` and `instructors` overlable into the article cards. This makes your website messy and therefore poor UI. Kindly consider fixing this in all the desktop screen sizes to ensure that your application is professional ☺️.
- [ ] In addition, the article cards should be vertically centered in the container and shouldn't overlap to other section of the application ☺️.

Your design 😎 
![image](https://github.com/s2jack/codewizards.github.io/assets/99394374/4495e15c-f12d-4a72-ad4e-0dfe8254f9ff)

Original design 😎 
![image](https://github.com/s2jack/codewizards.github.io/assets/99394374/14d996af-9801-4384-ba44-976143f220a8)